pvlib/test/test_ivtools.py

```diff
@@ -28,6 +28,13 @@ def get_cec_params_cansol_cs5p_220p():
                        'R_sh_ref': 837.51, 'R_s': 1.004, 'Adjust': 2.3}}
 
 
+@pytest.fixture
+def get_test_specs_params():
```

Comment:
I'd also change the fixture name to something more descriptive. For example, `fit_sdm_desoto_params`. While you'll sometimes see it, `get_` is superfluous in a fixture.